.gitlab-ci.yml

```diff
@@ -96,29 +96,26 @@ sniffers:sonarqube:
   - master
   allow_failure: true
 
-prepare:nodejs:
+prepare:front:
+  environment:
```

Comment:
I don't think we need environment variables here cause this isn' deployment job

Reply:
Good point, let me double check

Reply:
It was indeed not necessary, now removed